recipes/full_finetune_single_device.py

```diff
@@ -360,6 +362,10 @@ def _setup_model(
                 model, auto_wrap_policy={modules.TransformerSelfAttentionLayer}
             )
 
+        if quantizer_cfg is not None:
+            quantizer = config.instantiate(quantizer_cfg)
```

Comment:
it might be worth adding a logger here. Something like:

f"Quantizing the model for mixed precision training with precision {quantizer.precision}"